### ISSUE TITLE
Postgres backend

### ISSUE THREAD

**Issue by alysivji** [alysivji/busy-beaver#24]
Not really necessary, but it will be fun to practice switching over from SQLite to Postgres in production.

Need to set up data dumps to S3 or DO spaces

## Tasklist

- [x] move from SQLite to Postgres (#59)
- [x] set up backup strategy using CRON job
  - [x] create [pgdump](https://www.postgresql.org/docs/9.6/app-pgdump.html) script to output .sql file
  - [x] upload .sql file to s3 (boto adapter might be nice)
    - [x] set up s3 creds for busy-beaver (could use mine for now)
    - [x] upload using boto3 docker image
  - [x] update ansible config cron
	- [x] script to backup database and upload to s3
  - [x] have instructions on how to reload from backup
    - [x] practice doing it so it is second nature (make a script?)

**Comment by alysivji** [comment]
[backing up docker volumes](https://docs.docker.com/storage/volumes/#restore-container-from-backup)

**Comment by alysivji** [comment]
Also look into `pgdump`.

**Comment by alysivji** [comment]
I realized that the reason Busy Beaver could not create new accounts is because of an issue in moving databases from SQLite to Postgres. The autoincrement `id` field did not get set correctly so BB was trying to create a record for an `id` that already existed.

In order to fix the issue, I did a data dump and reran migrations on a fresh database. After reloading data, manually set the autoincrement field to the max value in the `id` column

Useful Resources 
- https://tech.willandskill.se/handling-postgres-auto-increment-field-errors-when-migrating-from-legacy-databases/
